.gitignore

```diff
@@ -6,29 +6,6 @@ __pycache__/
 # C extensions
 *.so
 
-# Specific intermediate files created by f2pỳ
-cflibformodule.c
-cflibfor-f2pywrappers2.f90
-
-# Benchs
-bench/
-
-# Files output by Gundam by default
```

Comment:
Let's keep these in the ignore file, too.

Reply:
Thanks, I added them back!